Gordon360/Services/HousingService.cs

```diff
@@ -111,11 +114,23 @@ public int SaveApplication(string editorId, string sess_cde, string [] applicant
             //----------------
             // Save hall information
 
-            // PLACEHOLDER
-            //
-            // The update hall info code will go here once we get a chance to implement it
-            //
-            // PLACEHOLDER
+            SqlParameter rankingParam = null;
+            SqlParameter buildingCodeParam = null;
+
+            foreach (ApartmentChoiceViewModel choice in apartmentChoices)
+            {
+                IEnumerable<ApartmentChoiceSaveViewModel> apartmentChoiceResult = null;
+
+                // All SqlParameters must be remade before being reused in an SQL Query to prevent errors
+                appIdParam = new SqlParameter("@APPLICATION_ID", apartAppId);
+                rankingParam = new SqlParameter("@RANKING", choice.HallRank);
+                buildingCodeParam = new SqlParameter("@BLDG_CDE", choice.HallName);
```

Comment:
My intention with naming that column BLDG_CDE was that it would have the same set of possible values that BLDG_CDE in Views/dbo.Buildings 
![image](https://user-images.githubusercontent.com/47014800/111207265-c8cb6800-859f-11eb-955b-6a0deb1c043c.png)
(That's just some example rows). 
We do not have to change it now because it would require either changing the table or creating a new stored procedure. Good to keep in mind, though.

Reply:
I think long term we should probably just change the apartment table to call that column `BUILDING_DESC` instead of `BLDG_CDE` since that is the form that we are using in every other place of the apartment application features. This info is only used to display the halls for the sake of the housing admin, so it seems unnecessary to add an additional conversion layer between the hall name and storing as the BLDG_CDE

If we do decide to use convert it to the BLDG_CDE values, it _should_ be a simple change, _in theory_